hs_core/hydroshare/resource.py

```diff
@@ -580,6 +580,9 @@ def create_new_version_resource(ori_res, new_res, user):
         # note that new version collection will not contain "deleted resources"
         new_res.resources = ori_res.resources.all()
 
+    # create the key/value metadata
+    new_res.extra_metadata = ori_res.extra_metadata
```

Comment:
This seems to be a shallow copy. So, the old version will be susceptible to changes in the new version???